controllers/metallb_controller.go

```diff
@@ -72,6 +72,7 @@ var EmbeddedFRRK8sSupportNotAvailable = errors.New("current CNO version does not
 // +kubebuilder:rbac:groups="",namespace=metallb-system,resources=configmaps,verbs=get;list;watch;create;update;patch;delete
 
 // Cluster Scoped
+// +kubebuilder:rbac:groups=apps,resources=deployments;daemonsets,verbs=list;watch
```

Comment:
why did we need this change?

Reply:
because it was needed :), it was complaining about without. 

From the commit
```

    Bump binaries to target 1.30 k8s version

    kind updated to have default version k8s 1.30

    operator-sdk v1.38.0 moves to k8s 1.30 API's and Kubebuilder v4 Scaffolding,

    RBAC changed because the client-go bump start failing with error
    ```
      pkg/mod/k8s.io/client-go@v0.30.3/tools/cache/reflector.go:232: failed to list *v1.Deployment: deployments.apps is forbidd
    en: User "system:serviceaccount:metallb-system:manager-account" cannot list resource "deployments" in API group "apps" at the cluster scope
    ```
```